.devcontainer/devcontainer.json

```
@@ -0,0 +1,53 @@
+{
+	"name": "Velocitas lib",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"runArgs": [
+		"--init",
+		"--privileged"
+	],
+	"containerEnv": {
+		"GITHUB_API_TOKEN": "${localEnv:GITHUB_API_TOKEN}"
+	},
+	"customizations": {
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				"python.pythonPath": "/usr/bin/python3",
+				"python.defaultInterpreterPath": "/usr/bin/python3",
+				// Only Flake8 is used as linter and static code analyzer, as faster tool
+				"python.linting.enabled": true,
+				"python.linting.flake8Enabled": true,
+				// Style Formatter
+				"python.formatting.provider": "black",
+				// Security Linter
+				"python.linting.banditEnabled": true,
+				"python.disableInstallationCheck": true,
+				"terminal.integrated.defaultProfile.linux": "zsh",
+				"terminal.integrated.profiles.linux": {
+					"zsh": {
+						"path": "/usr/bin/zsh"
+					}
+				}
+			},
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"dbaeumer.vscode-eslint",
+				"ms-azuretools.vscode-dapr",
+				"ms-azuretools.vscode-docker",
+				"ms-python.python",
+				"cschleiden.vscode-github-actions",
+				"rpdswtk.vsmqtt",
+				"dotjoshjohnson.xml",
+				"ms-kubernetes-tools.vscode-kubernetes-tools",
+				"matangover.mypy",
+				"augustocdias.tasks-shell-input",
+				"ms-python.isort",
+				"ms-python.flake8"
+			]
```

please check if we really need all the extensions
```suggestion
			"extensions": [
				"ms-python.python",
				"cschleiden.vscode-github-actions",
				"dotjoshjohnson.xml",
				"matangover.mypy",
				"ms-python.isort",
				"ms-python.flake8"
			]
```